analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.TryCatchFinally.cs

```diff
@@ -784,31 +784,33 @@ public void Catch_NestedInFinally()
     [TestMethod]
     public void CatchWhen_Simple()
     {
-        const string code = @"
-var tag = ""BeforeTry"";
-try
-{
-    Tag(""InTry"");
-}
-catch (Exception ex) when (ex is FormatException)
-{
-    tag = ""InCatch"";
-}
-finally
-{
-    tag = ""InFinally"";
-}
-tag = ""End"";";
+        const string code = """
+            var tag = "BeforeTry";
+            try
+            {
+                Tag("InTry");
+            }
+            catch (Exception ex) when (ex is FormatException)
+            {
+                tag = "InCatch";
+            }
+            finally
+            {
+                tag = "InFinally";
+            }
+            tag = "End";
+            """;
```

Comment:
You can revert this for consistency, all other test cases are aligning the code with the beginning of the line

Reply:
We improve these test cases CAYC style whenever we touch them.

Reply:
Oh ok, all good then